requirements-dev.txt

```diff
@@ -1,14 +1,14 @@
 black[jupyter] == 22.10.0
-isort == 5.10.1
+isort == 5.12.0
 jupyter
 mypy == 0.982
-nbconvert
+nbconvert>=7.2.9
 nbstripout == 0.6.1
 bump2version
-pre-commit == 2.20.0
-pytest
+pre-commit==3.0.4
+pytest==7.2.1
 pytest-cov
-pytest-docker
+pytest-docker==0.12.0
```

Comment:
Had to pin: v1.0 and beyond require a new fixture `docker_compose_command`, but after adding it, docker wouldn't be able to start the container because of networking issues 🤷🏽